src/components/Dropdown/components/Label/index.test.tsx

```diff
@@ -18,7 +18,7 @@
 
 import { PiAcorn } from 'react-icons/pi'
 
-import { DropdownItem, ItemLayout } from '../../props'
+import { DropdownItem, ItemLayout } from '../../Dropdown.props'
```

Comment:
don't add "Dropdown" to the file names, they already are in the Dropdown folder, we have decided to avoid this pattern as it simply clutters the file names

https://github.com/mia-platform/design-system/issues/685